bangazonapi/views/product.py

```diff
@@ -182,7 +187,14 @@ def update(self, request, pk=None):
 
         product_category = ProductCategory.objects.get(pk=request.data["category_id"])
         product.category = product_category
-        product.save()
+
+        try:
+            new_product.clean_fields(exclude='image_path')
+            new_product.save()
+        except ValidationError as ex:
+            return Response({"message": ex.args[0]}, status=status.HTTP_400_BAD_REQUEST)
+
+        # product.save()
```

Comment:
Looks like a comment was left in here. Otherwise well done!

Reply:
Ooops!  Forgot to nix that.  Done in new commit.